exercises/strings/countLetter/countLetter.js

```diff
@@ -23,13 +23,24 @@
 
 function countLetter(string, letter) {
   // This is your job. :)
+  let letter_count=[];
```

Comment:
Good job! This does what it should.

But rather than keep an ever-growing array and then counting its length, you can just keep track of the count directly, e.g.,

```js
function countLetter(string, letter) {
  let count = 0;

  for(let char of string) {
    if (char === letter) {
      count = count + 1;
    }
  }

  return count;
}
```